modules/apps/export-import/export-import-changeset-api/src/main/java/com/liferay/exportimport/changeset/portlet/action/ExportImportChangesetMVCActionCommandContributor.java

```diff
@@ -26,15 +26,10 @@
  * @author Máté Thurzó
  */
 @ProviderType
-public interface ExportImportChangesetMVCActionCommand
+public interface ExportImportChangesetMVCActionCommandContributor
```

Comment:
@RenanUmV The interface no longer needs to extend MVCActionCommand. It doesn't affect you extending the `~MVCActionCommand` in the impl class.

Reply:
@RenanUmV @rafaprax By the way, is `*Contributor` a good name? Usually this name is used by extension points that developers can provide their own implementation to provide/add/inject something into the current system. But here it's not doing that.